src/commands/admin/restrict.ts

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable no-empty-function */
```

Comment:
These probably shouldn't be here, I don't think anything breaks without them

Reply:
I meant to take those out I swear....